### ISSUE TITLE
Stylelint v14.16.1 False Error with NuxtJs v2

### ISSUE THREAD

**Issue by joshuadabejero** [stylelint/stylelint#6727]
### What steps are needed to reproduce the bug?

Hello, for reproduction:

1. `yarn create nuxt-app <project-name>`, `npx create-nuxt-app <project-name>`  or `npm init nuxt-app <project-name>`
2. `yarn dev` which run `stylelint \"**/*.{css,scss,sass,html,vue}\" --ignore-path .gitignore`

I believe this is a bug on stylelint end, node_modules/stylelint is having errors with NuxtJs v2 installation using yarn / npm.

![image](https://user-images.githubusercontent.com/29270980/226854884-e0218848-f50b-4238-ac89-5cf1786310a4.png)

What I did to fix, and found out that latest stylelint is having trouble was replace latest `node_modules/stylelint/` with `node_modules/stylelint/` which came from my old nuxt project with the same version of stylelint.

The configuration I will send below are the same with the old project and new project (which having error atm).

### What Stylelint configuration is needed to reproduce the bug?

```
module.exports = {
  customSyntax: 'postcss-html',
  extends: [
    'stylelint-config-standard',
    'stylelint-config-recommended-vue',
    'stylelint-config-prettier',
  ],
  // add your custom config here
  // https://stylelint.io/user-guide/configuration
  rules: {},
}
```

### How did you run Stylelint?

`"lint:js": "eslint --ext \".js,.vue\" --ignore-path .gitignore .",     "lint:style": "stylelint \"**/*.{css,scss,sass,html,vue}\" --ignore-path .gitignore",     "lint:prettier": "prettier --check .",     "lint": "yarn lint:js && yarn lint:style && yarn lint:prettier",     "lintfix": "prettier --write --list-different . && yarn lint:js --fix && yarn lint:style --fix",`

### Which version of Stylelint are you using?

`14.16.1`

### What did you expect to happen?

No errors on `yarn dev`, run as normal.

### What actually happened?

```
yarn run v1.22.1[9](https://github.com/devnotize/maasina-fe/actions/runs/4476463900/jobs/7892292441#step:7:10)
$ yarn lint:js && yarn lint:style && yarn lint:prettier
$ eslint --ext ".js,.vue" --ignore-path .gitignore .
$ stylelint "**/*.{css,scss,sass,html,vue}" --ignore-path .gitignore

layouts/error.vue
 40:1  ✖  Unknown rule selector-anb-no-unmatchable  selector-anb-no-unmatchable

1 problem (1 error, 0 warnings)

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Does the bug relate to non-standard syntax?

I am not sure.

### Proposal to fix the bug

I don't know how to fix this. I just copying node_modules at the moment so I can proceed at my current work task.

**Comment by ybiquitous** [comment]
@joshuadabejero Thanks for using the template. The `selector-anb-no-unmatchable` rule is available since [15.3.0](ttps://github.com/stylelint/stylelint/releases/tag/15.3.0). Could you update your Stylelint?

**Comment by joshuadabejero** [comment]
> @joshuadabejero Thanks for using the template. The `selector-anb-no-unmatchable` rule is available since [15.3.0](https://github.com/stylelint/stylelint/releases/tag/15.3.0). Could you update your Stylelint?

Hello, sorry how can I update Stylelint and what version I should use? 😅

**Comment by ybiquitous** [comment]
I'm not sure about your project, but I think you can update Stylelint through `package.json` or `yarn` command.

**Comment by joshuadabejero** [comment]
> I'm not sure about your project, but I think you can update Stylelint through `package.json` or `yarn` command.

I think the issue was not on stylelint but on Nuxt / Vuetify ends they probably not yet updating their packages with stylelint 15.3.0. I'm going to close the ticket now. Thank you @ybiquitous !
